UnicodeJsps/src/main/java/org/unicode/jsp/UnicodeUtilities.java

```diff
@@ -560,7 +563,8 @@ public static String getIdentifier(String script, boolean showDevProperties) {
                 showSet(
                         allowed,
                         showDevProperties,
-                        new CodePointShower("", "", showDevProperties, true, false, false),
+                        new CodePointShower(
```

Comment:
🚿

Reply:
For the longest time I thought that it actually meant 🚿 (the list-unicodeset tool has a tendency of showering the user with code points!).